pkg/workloads/containerd.go

```diff
@@ -237,7 +237,7 @@ func (c *containerDClient) processEvent(m EventMessage) {
 		stopIgnoringContainer(m.WorkloadID)
 		c.handleCreateWorkload(m.WorkloadID, true)
 	case EventTypeDelete:
-		Owner().DeleteEndpoint(endpointid.NewID(endpoint.ContainerIdPrefix, m.WorkloadID))
+		Owner().DeleteEndpoint(endpointid.NewID(endpointid.ContainerIdPrefix, m.WorkloadID))
```

Comment:
Looks like this is another change that would break `git bisect` because the endpointid refactor is in the other commit.